cypress/TestCases/FireboltCertification/SecureStorage.feature

```diff
@@ -224,4 +224,115 @@ Feature: SecureStorage
+            | Adding scope as device in 2 apps             | get stored value with scope as device and key as authTestTokenDevice    | update stored value for key authTestTokenDevice                | expected value for authTestTokenDevice stored data in securestorage  | clear stored value with scope as device  |
+
+    # 2nd app is launched
+    # set value1 in device scope
+    # validates the set values for 1st app
+    # set value2 in device scope
+    # validates the set values for 1st app
+    # validates the set values for 2nd app
+    # removes value1 with device scope
+    # validates the removed value for 1st app
+    # validates the removed value for 2nd app
+    # validates the remaining value for 1st app
+    # validates the remaining value for 2nd app
+    @Securestorage @coreSDK @sdk @transport
+    Scenario Outline: SecureStorage.remove - Positive Scenario: Validate <Scenario>
+        And 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        Given '3rd party app' invokes the 'Firebolt' API to '<Set_API_Key1>'
+        And 'Firebolt' platform responds with 'null for updating a secure data value'
+        And '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds with '<Validation_Key1>'
+        And '3rd party app' invokes the 'Firebolt' API to '<Set_API_Key2>'
+        And 'Firebolt' platform responds with 'null for updating a secure data value'
+        And '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        And 'Firebolt' platform responds with '<Validation_Key2>'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key1>'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key2>'
+        When '3rd party app' invokes the 'Firebolt' API to '<Remove_API_Key>'
+        Then 'Firebolt' platform responds with 'null for removing stored value'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key1>'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        Then 'Firebolt' platform responds with '<Validation_Key2>'
+        And 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key2>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key2>'
+
```

Comment:
if scope of remove is device, then if we remove in line 285 , then we should expect the same in 290 right ? basically the 2 steps in line 288 and 290 should be same right ?